network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -390,6 +395,21 @@ class ProductPage(FoundationMetadataPageMixin, Page):
         ),
     ]
 
+    def get_context(self, request, *args, **kwargs):
+        context = super().get_context(request, *args, **kwargs)
+        # In an effort to keep `product_page.html` changed as little as possible while there could be
+        # other work in other branches on that template, we're assigning ['product'] = self.
+        # TODO: At some point in the future, let's remove this and use `page.field_name` instead of
+        # using `product.field_name` in the template
+        context['product'] = self
```

Comment:
Currently we're using `product.value` instead of Wagtails `self.value` or `page.value` (which are the same in the Page context). I've opted to keep using `product.value` for the time being, though this should be replaced soon. If any doubt in review, I can swap out all the `product.value`s in the template for `page.value` and remove this line and the comments above it.